main.nf

```diff
@@ -7,24 +7,25 @@ nextflow.enable.dsl = 2
 
 include { MTBSEQ } from "./modules/mtbseq/mtbseq.nf"
 
-include { TBBWA } from './modules/mtbseq/tbbwa/tbbwa.nf' addParams(params.TBBWA)
-include { TBREFINE } from './modules/mtbseq/tbrefine/tbrefine.nf' addParams(params.TBREFINE)
-include { TBPILE } from './modules/mtbseq/tbpile/tbpile.nf' addParams(params.TBPILE)
```

Comment:
@Mxrcon , let's leave these params as part of the project as of now.

Reply:
but @abhi18av this params aren't defined and they're raising an error on the execution, because in `global_params` those params are blank

Reply:
I reverted this changes, but i think that we should track this error on a issue

Reply:
Then, let's try to use the following pattern

```nextflow
TBBWA {
// global params

// module params

// <- HERE WE ADD THE MODULE LEVEL PARAMS MENTIONED IN tbbwa.nf file ->
}


```